debian/indi-ssag/changelog

```diff
@@ -1,3 +1,9 @@
+indi-ssag (1.0-1) buster; urgency=medium
+
+  * Changing dependency to libasi
```

Comment:
Why SSAG depends on asi?

Reply:
Originally it depends on asi-common, which will be replaced by libasi. I don't know where the original dependency come from.

Reply:
It think it was there by mistake, so this dependency needs to be removed.

Reply:
Done